src/image.c

```diff
@@ -51,6 +55,63 @@ void image_rotate(struct image* ctx, size_t angle)
     }
 }
 
+// TODO: probably should be moved to config, to make use of expand_path()
+/**
+ * Get path to cached image thumbnail.
+ * @param image image
+ * @param path array in which the path should be stored
+ * @return true if successful
+ */
+static bool get_image_thumb_path(struct image* image, char* path)
+{
+    static char* cache_dir = NULL;
+    int r;
+    if (!cache_dir) {
+        cache_dir = getenv("XDG_CACHE_HOME");
+        if (cache_dir) {
+            cache_dir = strcat(cache_dir, "/swayimg");
```

Comment:
Wow =)

Reply:
Would it be fine to put `export_path()` in config.h?
Then I could use it here

Reply:
Sorry, I meant `expand_path()`.

Reply:
This function is specific to thumbnails, so leave it in `thumbnail.c`.

Reply:
Let me rephrase the question:
Would it be alright to add `expand_path()` to config.h header file?
It is currently implemented as a static function in config.c:203

Reply:
Ah, I get it =)
Yes, it will be ok.